### PR TITLE
Feature/attendance pages

### PR DESCRIPTION
**Pull request by AyaobaEstelle**

### CODE REVIEW

File: src/app/create/special-calendar/page.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"use client";
```
Comment (Luwat) [inline src/app/create/special-calendar/page.tsx:1]: I don't think you need to 'use client' for this document.

**Review by Luwat — COMMENTED** [review]
Well done @AyaobaEstelle.

**Comment by Luwat** [comment]
Your commit messages should be in present tense, the first letter of the first word should be capitalised and should be instructions e.g "Install react-icons" not "installed react-icons"

File: src/app/create/special-calendar/page.tsx (hunk 1 above)
Comment (AyaobaEstelle) [reply src/app/create/special-calendar/page.tsx:1]: i think i was a mistake ill  fix it

**Comment by AyaobaEstelle** [comment]
okay got it @Luwat  thank you

**Pull request merged**